bzt/engine.py

```diff
@@ -577,7 +578,11 @@ def load(self, config_files, callback=None):
                 configs = []
                 self.log.debug("Reading %s", config_file)
                 with open(config_file) as fds:
-                    configs.extend(yaml.load_all(fds))
```

Comment:
isn't `load_all` accept str/bytes as argument? Their doc demonstrates that http://pyyaml.org/wiki/PyYAMLDocumentation